packages/conf-gmp/conf-gmp.4/opam

```diff
@@ -16,6 +16,11 @@ build: [
     "cc -c $CFLAGS -I/opt/homebrew/include -I/opt/local/include -I/usr/local/include test.c"
   ] {os = "macos"}
```

Comment:
Maybe add a similar test for `os = "win32" & os-distribution = "cygwin"` with `host-arch-x86_64:installed` and `host-arch-x86_32:installed`?
If you use `$(ocamlc -config-var c_compiler)`, you don't even have to add include directories as the MinGW compiler should already use the one where `conf-mingw-...` installed its files.

Reply:
fixed